docs/hub/enterprise-hub-gating-group-collections.md

```diff
@@ -0,0 +1,166 @@
+
+<div class="flex justify-center" style="max-width: 750px">
+    <img
+        class="block dark:hidden !m-0"
+        src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/collections/gating-group-repo-settings.webp"
+        alt="Hugging Face repo access settings when repo is in a gating group collection"
+    />
+    <img
+        class="hidden dark:block !m-0"
+        src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/collections/dark-gating-group-repo-settings.webp"
+        alt="Hugging Face repo access settings when repo is in a gating group collection"
+    />
+</div>
+
+This will open a modal with 3 lists of users:
+
+- **pending**: the list of users waiting for approval to access your repository. This list is empty unless you’ve selected **Manual Review**. You can either **Accept** or **Reject** the demand. If the demand is rejected, the user cannot access your repository and cannot request access again.
+- **accepted**: the complete list of users with access to your repository. You can choose to **Reject** access at any time for any user, whether the approval mode is manual or automatic. You can also **Cancel** the approval, which will move the user to the **pending** list.
+- **rejected**: the list of users you’ve manually rejected. Those users cannot access your repositories. If they go to your repository, they will see a message _Your request to access this repo has been rejected by the repo’s authors_.
+
+<div class="flex justify-center" style="max-width: 750px">
+    <img 
+        class="block dark:hidden"
+        src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/models-gated-enabled-pending-users.png"
+        alt="Manage access requests modal for a repo in a gating group collection"
+    />
+    <img
+        class="hidden dark:block"
+        src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/models-gated-enabled-pending-users-dark.png"
+        alt="Manage access requests modal for a repo in a gating group collection"
+    />
+
+</div>
+
+#### Via the API
+
+You can programmatically manage access requests in a Gated Group Collection through the API of any of its models or datasets.
+
+Visit our [gated models](https://huggingface.co/docs/hub/models-gated#via-the-api) or [gated datasets](https://huggingface.co/docs/hub/datasets-gated#via-the-api) documentation to know more about it.
+
+#### Download access report
+
+You can download access reports for the Gated Group Collection through the settings page of any of its models or datasets.
+
+Visit our [gated models](https://huggingface.co/docs/hub/models-gated#download-access-report) or [gated datasets](https://huggingface.co/docs/hub/datasets-gated#download-access-report) documentation to know more about it.
+
+#### Customize requested information
+
+<Tip warning={true}>
```

Comment:
I'd maybe suggest to reorder this section. We can start by simply mentioning that gating parameters and requested information can be customized, then linking to the docs, then putting the tip at the end.